packages/@react-aria/textfield/src/useTextField.ts

```diff
@@ -65,9 +66,25 @@ export function useTextField(
     pattern: props.pattern
   };
 
+  // TODO: should we support press events on textfield?
+  // This is added here so that pressing on a textfield in a table properly focuses the textfield
+  // and that selection doesn't trigger since usePress will stop propagation. However, usePress prevents default
+  // and thus stops text selection via mouse from working....
+  // let {pressProps} = usePress({
+  //   ref
+  // });
+  // The below is an alternative that allows the user to click into a textfield in a editable cell
+  let onPointerDown = (e) => {
+    ref.current.focus();
+    e.stopPropagation();
+  };
+
+
```

Comment:
Kinda iffy on this. If we don't manually focus the textfield on pointer down, clicking on a textfield in a tableview won't move focus into it. Buttons and other elements don't have this issue because they have `usePress` which manually focuses the element on pointer down, but we don't really want to use `usePress` in textfields because it will preventDefault and stop text selection via mouse.

Open to opinions on this, personally I want to have this focus behavior so that the user can do the following cases:
- click into a textfield in a tableview without needing to activate edit mode on the cell via keyboard first (this is consistent with how other components like buttons behave in a tableview right now)
- move focus between textfields in a tableview via clicking